app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -220,6 +226,49 @@ class PatientSummaryScreen :
     )
   }
 
+  override fun defaultModel(): PatientSummaryModel {
+    return PatientSummaryModel.from(screenKey.intention, screenKey.patientUuid)
+  }
+
+  override fun bindView(layoutInflater: LayoutInflater, container: ViewGroup?): ScreenPatientSummaryBinding {
+    return ScreenPatientSummaryBinding.inflate(layoutInflater, container, false)
+  }
+
+  override fun uiRenderer(): ViewRenderer<PatientSummaryModel> {
+    return PatientSummaryViewRenderer(this)
+  }
```

Comment:
Not necessary to change this, but adding that I prefer to convert these to single line expression functions which reduces a lot of the verbosity.

Reply:
Good point. Will start doing that for future migrations.